### PR TITLE
Additional callings support

### PR DESCRIPTION
**Pull request by ntowle**
- Adds tooltips to callings
- Fix for multi-line callings (didn't display before)

### CODE REVIEW

File: SavedInstances/Core/Core.lua [hunk 1]
```diff
@@ -1816,6 +1816,46 @@ hoverTooltip.ShowEmissaryTooltip = function (cell, arg, ...)
...
+  text = info.title
+  if not text then
+    for _, t in pairs(SI.db.Toons) do
+      if t.Calling and t.Calling[day] and t.Calling[day].title then
+        text = t.Calling[day].title
+        break
+      end
+    end
+  end
+  indicatortip:SetCell(2, 1, text, "LEFT", 2)
```
Comment (LiangYuxuan) [inline SavedInstances/Core/Core.lua:1849]: `text` might be nil here, use `L["Calling Missing"]`

File: SavedInstances/Modules/Calling.lua [hunk 2]
```diff
@@ -71,6 +73,16 @@ function Module:COVENANT_CALLINGS_UPDATED(_, callings)
         t.Calling[day].isFinished = isFinished
         t.Calling[day].questDone = questDone
         t.Calling[day].questNeed = questNeed
+
+        local numQuestRewards = GetNumQuestLogRewards(data.questID)
+        if numQuestRewards > 0 then
+          local itemName, _, _, quality = GetQuestLogRewardInfo(1, data.questID)
+          if itemName then
+            t.Calling[day].questReward = {}
```
Comment (LiangYuxuan) [inline SavedInstances/Modules/Calling.lua:81]:
```lua
t.Calling[day].questReward = t.Calling[day].questReward and wipe(t.Calling[day].questReward) or {}
```
to reduce table creating

**Review by LiangYuxuan — CHANGES_REQUESTED** [review]
Thank you! It's good job but have some minor issue to resolve :)

Also, would you put your new `.questReward` to DB structure commend in Core.lua.

https://github.com/SavedInstances/SavedInstances/blob/64e7b2cc56737023e88bdbe8c972e24f57059a78/SavedInstances/Core/Core.lua#L241-L254

**Review by LiangYuxuan — APPROVED** [review]

**Review by Stanzilla — APPROVED** [review]
That lgtm, thank you!

**Pull request merged**